src/App.vue

```diff
@@ -309,37 +322,43 @@ export default {
   },
 
   methods: {
-    ...mapActions([
-      'loadBusinessInfo',
-      'loadFilings',
-      'loadStateFiling',
-      'setAuthRoles',
-      'setBusinessAddress',
-      'setBusinessEmail',
-      'setBusinessPhone',
-      'setBusinessPhoneExtension',
-      'setConfigObject',
-      'setCorpTypeCd',
-      'setCurrentDate',
-      'setCurrentJsDate',
-      'setEntityStatus',
-      'setFilings',
-      'setKeycloakRoles',
-      'setNameRequest',
-      'setParties',
-      'setRecordsAddress',
-      'setRegisteredAddress',
-      'setTasks',
-      'setUserKeycloakGuid'
-    ]),
-
-    ...mapMutations([
-      'mutateFetchingDataSpinner',
-      'setLegalName',
-      'setLegalType',
-      'setGoodStanding',
-      'setIdentifier'
-    ]),
+    ...mapActions(useBusinessStore,
+      [
+        'loadBusinessInfo',
+        'setLegalName',
+        'setLegalType',
+        'setGoodStanding',
+        'setIdentifier'
+      ]),
```

Comment:
Could you take a pass at alphabetizing all the getters and setters, please? (Easier to find when coding or reviewing code.)

Reply:
I added this as a "todo" in bcgov/entity#16207.